docs/secure-connections/sauce-connect/proxy-tunnels.md

```diff
@@ -23,37 +23,56 @@ The Tunnels page displays useful information, such as the number of active tunne
 
-### Starting a New Tunnel via Command Line
+## Starting Tunnels
+Every Sauce Connect Proxy tunnel spins up a fresh virtual machine (VM) that is used only for your tests. Once the tunnel is closed, VMs are destroyed. As a best practice, we recommend creating a new tunnel for each test suite or build and tearing it down at the end of your test. For information about user roles and permissions, see [User Roles](/basics/acct-team-mgmt/managing-user-info).
+
+
+### From the Command Line
 You can launch a new tunnel from the command line of the machine where the Sauce Connect Proxy client is installed by copying the **Run Command**, which will include your authentication credentials. You can also add any Sauce Connect Proxy parameters you want to use in configuring your tunnel.
 
 See [Basic Setup](/secure-connections/sauce-connect/setup-configuration/basic-setup) for full instructions on launching tunnels.
 
-### Stopping a Tunnel via Command Line
-There are two ways to do this:
 
-#### Method 1: `ctrl-c`
-Once Sauce Connect Proxy has been terminated (typically via `ctrl-c`), a call will be made from Sauce Connect to the REST API with instructions to terminate the Tunnel VM. Sauce Connect will continue to poll the REST API until the Tunnel VM has been halted and deleted.
+### From the Sauce Labs UI
+Currently, this is not supported.
+
+
+
+## Stopping Tunnels
+
+### From the Command Line
+
+#### To Stop a Single Tunnel: `Ctrl+C`
+Enter `Ctrl+C` to terminate your tunnel. A call will be made from Sauce Connect to the REST API with instructions to terminate the Tunnel VM. Sauce Connect will continue to poll the REST API until the Tunnel VM has been halted and deleted.
+
+```bash
+^C
+Stopping client
+Will wait for up to 300s for any active jobs using this tunnel to finish.
+Note: Press CTRL-C again to shut down immediately.
+Note: If you do this, tests that are still running will fail.
+Waiting for the connection to terminate...
+Connection closed (8).
+Goodbye.
+```
 
 :::note
 If you attempt to terminate a Sauce Connect Proxy tunnel that is running a test with `ctrl-c`, you will see a message indicating that Sauce Connect Proxy will not terminate until tests are completed. To proceed with terminating Sauce Connect Proxy before the test finishes, enter `ctrl-c` again to force it to quit.
 :::
 
-#### Method 2: `KILL` signal
-To stop an individual tunnel via the command line/prompt, you must send some sort of `KILL` signal to the running `Process ID` (pid).
-
-1. Start the Sauce Connect Proxy process.
-  ```bash
-  $ ./sc-<VERSION>-<PLATFORM>/bin/sc -u $SAUCE_USERNAME -k $SAUCE_ACCESS_KEY
-  ```
+#### To Stop a Single Tunnel: `KILL` signal
+Another way to stop an individual tunnel via command line is to send a `KILL` signal to the running `Process ID` (pid).
```

Comment:
The `kill` command sends various signals. `kill -2` sends a `SIGINT` signal, which interrupts the program for a graceful shutdown. `kill -9` sends a kill signal.

For this example, changing `KILL` to `SIGINT` should work.